packages/teams-js/src/private/hostEntity/tab.ts

```diff
@@ -105,6 +112,12 @@ class HostEntityTabInstancesResponseHandler extends ResponseHandler<HostEntityTa
     return response;
   }
 }
+export function addAndConfigure(hostEntityIds: HostEntityIds, appTypes?: AppTypes[]): Promise<HostEntityTabInstance>;
```

Comment:
You'll need type JSDoc comments on these definitions for them to documented correctly